Source/Scene/Scene.js

```diff
@@ -793,6 +814,142 @@ define([
+        scene._copyDepthFramebuffer = scene._copyDepthFramebuffer && !scene._copyDepthFramebuffer.isDestroyed() && scene._copyDepthFramebuffer.destroy();
+    }
+
+    function createTextures(scene, context, width, height) {
+        destroyTextures(scene);
+
+        scene._colorTexture = context.createTexture2D({
+            width : width,
+            height : height,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.UNSIGNED_BYTE
+        });
+        scene._colorTexture.sampler = context.createSampler({
+            minificationFilter : TextureMinificationFilter.NEAREST,
+            magnificationFilter : TextureMagnificationFilter.NEAREST
+        });
+
+        scene._depthStencilTexture = context.createTexture2D({
+            width : width,
+            height : height,
+            pixelFormat : PixelFormat.DEPTH_STENCIL,
+            pixelDatatype : PixelDatatype.UNSIGNED_INT_24_8_WEBGL
+        });
+        scene._depthStencilCopyTexture = context.createTexture2D({
+            width : width,
+            height : height,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.FLOAT
+        });
+    }
+
+    function createFramebuffers(scene, context) {
+        destroyFramebuffers(scene);
+
+        scene._framebuffer = context.createFramebuffer({
+            colorTextures : [scene._colorTexture],
+            depthStencilTexture : scene._depthStencilTexture
+        });
+
+        scene._copyDepthFramebuffer = context.createFramebuffer({
+            colorTextures : [scene._depthStencilCopyTexture]
+        });
+
+        var complete = WebGLRenderingContext.FRAMEBUFFER_COMPLETE;
+        if (scene._framebuffer.status !== complete || scene._copyDepthFramebuffer.status !== complete) {
+            destroyFramebuffers(scene);
+            return false;
+        }
+
+        return true;
+    }
+
+    function updateFramebuffers(scene) {
+        var context = scene._context;
+        if (!context.depthTexture) {
+            return;
+        }
+
+        var width = context.drawingBufferWidth;
+        var height = context.drawingBufferHeight;
+
+        var colorTexture = scene._colorTexture;
+        var textureChanged = !defined(colorTexture) || colorTexture.width !== width || colorTexture.height !== height;
+        if (textureChanged) {
+            createTextures(scene, context, width, height);
+        }
+
+        if (!defined(scene._framebuffer) || textureChanged) {
+            if (!createFramebuffers(scene, context)) {
+                // framebuffer creation failed
+                return;
+            }
+        }
+
+        context.uniformState.globeDepthTexture = scene._depthStencilCopyTexture;
+
+        if (!defined(scene._oit) && scene._useOIT) {
+            scene._oit = new OIT(context, scene._framebuffer);
+        }
+    }
+
+    function updateCopyCommands(scene) {
```

Comment:
I think we are jamming too much into Scene.js.  The post-process framework would help, but in the meantime what do you think of:
   * Abstract away the framebuffer/texture management (creation, resize, etc.) into a separate private class like I started in the [postprocess work](https://github.com/AnalyticalGraphicsInc/cesium/compare/postprocess-hook).
   * Move the copy commands and friends into their own private primitive/pass/whatever kinda like `SkyBox`

This might wind up being one class, not two.  I dunno.  See what fits.